src/scripts/main.js

```diff
@@ -1,3 +1,77 @@
 'use strict';
 
-// write code here
+const appendRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const appendColButton = document.querySelector('.append-column');
+const removeColButton = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
+
+const minCount = 2;
+const maxCount = 10;
+let currentRowCount = table.rows.length;
+let currentColCount = table.rows[0].cells.length;
+
+appendRowButton.addEventListener('click', () => {
+  if (currentRowCount < maxCount) {
```

Comment:
Use early return to simplify nested if statements:
```suggestion

  if (currentRowCount >= maxCount) {
    return;
  }
```